src/Evolution/Executables/Burgers/EvolveBurgers.hpp

```diff
@@ -82,12 +86,23 @@ struct EvolutionMetavars {
       tmpl::conditional_t<local_time_stepping,
                           dg::Actions::ApplyBoundaryFluxesLocalTimeStepping,
                           tmpl::list<>>,
-      Actions::UpdateU>>;
+      Actions::UpdateU,
+      // TODO FOR CODE REVIEW:
+      // Triggering the limiting is likely beyond the intended scope of
+      // update_variables. Because limiting should be applied after time steps
+      // in both the evolution and the self-start procedure, adding the actions
+      // here does the job. Should discuss possibility of a better design.
+      SlopeLimiters::Actions::SendData<EvolutionMetavars>,
+      SlopeLimiters::Actions::Limit<EvolutionMetavars>>>;
 
   struct EvolvePhaseStart;
   using component_list = tmpl::list<DgElementArray<
       EvolutionMetavars, dg::Actions::InitializeElement<1>,
       tmpl::flatten<tmpl::list<
+          // Optionally, one may wish to limit the initial data. This is not
```

Comment:
I would just make an issue for this to examine later.  Once we have action lists per phase this will be easy to do.